lib/themes/dosomething/paraneue_dosomething/js/components/FeatureCard.js

```diff
@@ -188,7 +201,7 @@ class FeatureCard extends React.Component {
               <h2 className="heading -delta">{campaign.title}</h2>
               <p className="tagline">{campaign.tagline}</p>
               <ul className="form-actions">
-                {(campaign.signedUp && !this.state.isSignedUp) ? <h1>You're signed up already!</h1> : <li><button className={`${this.state.isSignedUp ? 'tada' : ''} button`} onClick={()=>this.signup()}>Sign Up</button></li>}
+                <li><Button loading={this.state.isLoading} success={this.state.isSignedUp} action={this.signup}></Button></li>
```

Comment:
It might be cleaner to bind all the event handlers to the component instance in the constructor (like [this](https://github.com/DoSomething/voting-app/blob/04234aebff2cf60114e3d0c36dd02d25d2fe555b/resources/assets/js/components/Tile.js#L10)), but either way should work fine.